### PR TITLE
New Release

### PR DESCRIPTION
**Pull request by janstaelensskyline**

### CODE REVIEW

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.0.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ uid: Skyline_DataMiner_Core_DataMinerSystem_Range_1.0
 > [!NOTE]
 > Range 1.0.x.x is supported as from **DataMiner 10.1.0**. This is a continuation from the now obsolete [Class Library](xref:ClassLibrary_Range_1.2).
 
+### 1.0.1.4
+
+#### Fix - 1 minute timeout on Start and Stop Monitors increased to 10 minutes
+
+When starting or stopping a Monitor, creating or clearing the underlying subscription could sometimes time-out after 1 minute on very loaded DataMiner systems.
+
+We increased the timeout to 10 minutes and also improved the exception logging to make it more clear to the user that this was an SLNet Problem.
```
Comment (MichielOda) [inline release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.0.md:16]:
```suggestion
We increased the timeout to 10 minutes and also improved the exception logging to make it more clear to the user that this was a SLNet Problem.
```

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.1.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ uid: Skyline_DataMiner_Core_DataMinerSystem_Range_1.1
 > [!NOTE]
 > Range 1.1.x.x is supported as from **DataMiner 10.1.11**. It makes use of a change introduced in DataMiner 10.1.11 that makes it possible to obtain table cell data using the primary key. In earlier DataMiner versions, the display key was needed to obtain this data.
 
+### 1.1.1.4
+
+#### Fix - 1 minute timeout on Start and Stop Monitors increased to 10 minutes
+
+When starting or stopping a Monitor, creating or clearing the underlying subscription could sometimes time-out after 1 minute on very loaded DataMiner systems.
+
+We increased the timeout to 10 minutes and also improved the exception logging to make it more clear to the user that this was an SLNet Problem.
```
Comment (MichielOda) [inline release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.1.md:16]:
```suggestion
We increased the timeout to 10 minutes and also improved the exception logging to make it more clear to the user that this was a SLNet Problem.
```

**Review by MichielOda — CHANGES_REQUESTED** [review]

File: release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.0.md (hunk 1 above)
Comment (MariekeGO) [reply release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.0.md:16]: It should in fact be "an SLNet Problem". Whether you use "a" or "an" is determined by the way you pronounce the word that follows, not by the way that word is spelled. "SLNet" sounds as if it starts with a vowel, so "an" should be used.

Comment (MichielOda) [reply release-notes/Code_Libraries/Skyline.DataMiner.Core/Skyline.DataMiner.Core.DataMinerSystem/Skyline_DataMiner_Core_DataMinerSystem_Range_1.0.md]: Thanks for the clarification!

**Review by spetebroodt — APPROVED** [review]

**Pull request merged**